airflow/cli/cli_config.py

```diff
@@ -0,0 +1,2250 @@
+#!/usr/bin/env python
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Command-line interface."""
```

Comment:
Can we use docstrings of both modules to better describe what they do?

(As a separate issue, calling the modules `cli_parser` and `cli_config` seems redundant since they are already in a `cli` subpackage. It’s probably difficult to rename `cli_parser`, but we can name the new module `config`.)

Reply:
> Can we use docstrings of both modules to better describe what they do?

Sure, good suggestion, I'll add those.

>(As a separate issue, calling the modules cli_parser and cli_config seems redundant since they are already in a cli subpackage. It’s probably difficult to rename cli_parser, but we can name the new module config.)

As you said `cli_parser.py` is difficult to change at this point and given that, I think it's cleaner to have both with the same naming format.

Reply:
> I think it's cleaner to have both with the same naming format.

We already have `airflow.cli.simple_table` and `airflow.cli.command`, I’d argue _not_ having the `cli_` prefix is the preferred format.

Reply:
I have also seen issues in the past with module names that are too generic. You could have many `config` modules, and if you happen to need more than one as dependencies in another module you end up renaming them anyway on the import `from airflow.cli import config as cli_config`. I don't think that super likely here, but I generally try to avoid overly generic module names.

Reply:
Added doc strings (marking as resolved)

Reply:
Yeah. I agree generic module names are a recipe for problems.